Specs/Scene/SceneSpec.js

```diff
@@ -251,6 +251,9 @@ defineSuite([
     });
 
     it('debugCommandFilter does not filter commands', function() {
+        var originalLogDepth = scene.logarithmicDepthBuffer;
```

Comment:
Does this file or `MultifrustumSpec.js` need some new tests where it switches between log depth on/off and sees the number of (redundant) commands executed?

In general, how is code coverage?  Does anything need more fine-grained tests?  Shouldn't the new built-in GLSL functions have new tests?

Reply:
The multi-frustum tests have been updated to disable log depth. One test has it enabled to show that log depth has less 1 frustum where the other had 3.

Code coverage seems ok, but it's hard to say. I can't add tests for the new GLSL built-ins since they don't return a value. They update gl_Position and gl_FragDepth.